appveyor.yml

```diff
@@ -14,26 +14,24 @@ environment:
     PYTHONUNBUFFERED: 1
 
   matrix:
-    - PYTHON: "C:\\Miniconda"
-      CONDA_PY: "27"
-      ARCH: '32'
-    - PYTHON: "C:\\Miniconda-x64"
-      CONDA_PY: "27"
-      ARCH: '64'
-    - PYTHON: "C:\\Miniconda36"
-      CONDA_PY: "36" # No MDTraj for py37/Win32
-      ARCH: '32'
+    #- PYTHON: "C:\\Miniconda38-x64"
+      #CONDA_PY: "38" 
+      #ARCH: '64'
     - PYTHON: "C:\\Miniconda37-x64"
       CONDA_PY: "37"
       ARCH: '64'
+    - PYTHON: "C:\\Miniconda36-x64"
+      CONDA_PY: "36" 
+      ARCH: '64'
 
 build: false
 
 install:
   # install python
-  - SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%
+  - SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PYTHON\\bin;%PATH\\Library\\bin;%PATH%
   - conda config --add channels conda-forge
-  - conda install -yq conda=4.5 conda-build=3.10
+  #- conda install -yq conda=4.5 conda-build=3.10
```

Comment:
please remove the old (I presume temporary) install command. Or do you think we might need it in the future?

Reply:
done